docs/install.rst

```diff
@@ -0,0 +1,352 @@
+
+    # Register with the CrateDB package repository.
+    sudo rpm -Uvh https://cdn.crate.io/downloads/yum/7/x86_64/crate-release-7.0-1.x86_64.rpm
+
+The command above will install the ``/etc/yum.repos.d/crate.repo`` package
+repository configuration file.
+
+.. NOTE::
+
+    CrateDB provides both *stable release* and *testing release* channels. You
+    can read more about the `release workflow`_.
+
+    By default, yum_ (Red Hat's package manager) will use the stable
+    repository. This is because the testing repository is disabled.
+    If you would like to enable the testing repository, edit the ``crate.repo``
+    file and set ``enabled=1`` within the ``[crate-testing]`` section.
+
+
+Install CrateDB
+---------------
+
+With everything set up, you can install CrateDB::
+
+    sudo yum install crate
+
+After the installation is finished, the ``crate`` service should be installed,
+but not running. Use the following command to start CrateDB::
+
+    sudo systemctl start crate
+
+In order to make the service reboot-safe, invoke::
+
+    sudo systemctl enable crate
+
+
+Control CrateDB
+---------------
+
+You can control the ``crate`` service with the ``systemctl`` utility program::
+
+    sudo systemctl COMMAND crate
+
+Replace ``COMMAND`` with ``start``, ``stop``, ``restart``, ``status`` and
+so on.
+
+
+Docker
+======
+
+CrateDB and Docker_ are great matches thanks to CrateDB's shared-nothing,
+horizontally scalable architecture that lends itself well to containerization.
+
+In order to spin up a container using the most recent stable version of the
+official `CrateDB Docker image`_, use::
+
+    docker run --publish=4200:4200 --publish=5432:5432 crate
+
+.. TIP::
+
+    If this command aborts with an error, please consult the `Docker
+    troubleshooting guide`_. You are also welcome learn more about
+    :ref:`howtos:resource_constraints` with respect to running CrateDB within
+    containers.
+
+.. CAUTION::
+
+    This type of invoking CrateDB will get you up and running quickly.
+
+    Please note, by default, the CreateDB Docker container is ephemeral, so
+    data will not be stored in a persistent manner. When stopping the
+    container, all data will be lost.
+
+    When you are ready to start using CrateDB for data you care about, please
+    consult the :ref:`full guide to CrateDB and Docker <howtos:cratedb-docker>`
+    in order to configure the Docker setup appropriately by using persistent
+    disk volumes.
+
+
+.. _configure:
+
+Basic configuration
+===================
+
+In order to configure CrateDB, take note of the configuration file location and
+the available environment variables.
+
+
+Configuration files
+-------------------
+
+The main CrateDB configuration files are located within the ``/etc/crate``
+directory.
+
+
+Environment variables
+---------------------
+
+The CrateDB startup script uses :ref:`reference:conf-env` from the
+``/etc/default/crate`` file. Here is an example::
```

Comment:
to be fair the path is only correct for the packaged versions (deb,yum)

env variables in general are valid either way

Reply:
I extended this sentence to also include the correct path for the "ad-hoc installation".